src/wrapper.js

```diff
@@ -20,19 +20,29 @@ function accessibleAutocomplete(options) {
 
   renderer = render(<Autocomplete {...options} />, options.element);
 
+  if (typeof options.selectElement === 'undefined') {
+    return renderer;
+  }
+
   // create MutationObserver and re-render <Autocomplete />
   observer = new MutationObserver(mutationsList => {
     for (const mutation of mutationsList) {
       if (mutation.type == 'childList') {
         options.source = createSimpleEngine(getSourceArray(options));
         render(<Autocomplete {...options} />, options.element, renderer);
       }
+      if (mutation.type == 'attributes') {
+        options.inputClassName = mutation.target.classList;
+        render(<Autocomplete {...options} />, options.element, renderer);
```

Comment:
Jemne sa mi nepacia ze mame 2 renderi vo funkcii. 

Neviem ci by sme nemali radsej volat render az za for cyklom? Ci je to blbost?